pattern-1/dockerfiles/integrator/build.sh

```diff
@@ -31,4 +33,4 @@ tag=${productVersion}
 
 echo "Creating ${profile} profile for pattern ${patternNumber} ..."
```

Comment:
Shall we use the same, echo "Creating ${repository}:${tag}..." here as well ?

Reply:
+1 Thanks for pointing! Will add that!